docs/02-Install.md

```diff
@@ -88,41 +87,67 @@ Under the `Study` section:
  * `ImagingUploader Auto Launch`: Used by the Imaging Uploader to automatically launch the insertion scripts on the uploaded scan
  
 Under the `Paths` section: 
- * `Imaging Data`: Where the imaging data is stored; typically `/data/$PROJECT/data/`
  * `LORIS-MRI Code`: Where the MRI code base is installed; typically `/data/$PROJECT/bin/mri/`
  * `MRI-Upload Directory`: Where the uploaded scans get stored; typically `/data/incoming/`
- * `MINC Files`: Where the MINC images are stored; typically `/data/$PROJECT/data/`
  * `Images`: Where the images displayed in Imaging Browser are stored; typically `/data/$PROJECT/data/`
  
- Under the `Imaging Modules` section: 
+Under the `Imaging Modules` section: 
   * `Patient ID regex`: Used by the DICOM Archive module to show/hide the PatientID info
   * `Patient name regex`: Used by the DICOM Archive module to show/hide the Patient Name info
   * `Lego phantom regex`: Used by the DICOM Archive module to show/hide the Patient Name info for phantoms
   * `Living phantom regex`: Used by the DICOM Archive module to show/hide the Patient Name info for phantoms
   * `Imaging Browser Tabulated Scan Types`: Used by Imaging Browser's main page which lists the different imaging sessions across candidates. This setting will determine which modalities will have their QC status displayed in that listing page
      
- Under the `Imaging Pipeline` section: 
- * `LORIS-MRI Data Directory`: Directory where imaging data is stored; typically `/data/$PROJECT/data/`
- * `Study Name`: Prefix to be used in all filenames inserted into the `files` table and visible in the front-end via the Imaging Browser module
- * `User to notify when executing the pipeline`: User email address to be used when notification is to be sent by the pipeline
- * `Full path to get_dicom_info.pl script`: Typically `/data/$PROJECT/bin/mri/dicom-arhive/get_dicom_info.pl`
- * `Horizontal pictures creation`: Used to pass or not pass the argument `-horizontal` to `mincpik` when generating pictures to be displayed in Imaging Browser. 
+Under the `Imaging Pipeline` section: 
+ * `LORIS-MRI Data Directory`: Directory where imaging data is stored; 
+    typically `/data/$PROJECT/data/`
+ * `Study Name`: Prefix to be used in all filenames inserted into the `files` 
+    table and visible in the front-end via the Imaging Browser module
+ * `User to notify when executing the pipeline`: User email address to be used when 
+    notification is to be sent by the pipeline
+ * `Full path to get_dicom_info.pl script`: typically `/data/$PROJECT/bin/mri/dicom-arhive/get_dicom_info.pl`
+ * `Horizontal pictures creation`: Used to pass or not pass the argument `-horizontal` 
```

Comment:
I think this sentence should be changed: 

specifies whether or not argument `-horizontal` should be used by `mincpik` when generating pictures to be displayed in Imaging Browser.